cli/compat/mod.rs

```diff
@@ -125,3 +125,21 @@ pub(crate) fn add_global_require(
 fn escape_for_single_quote_string(text: &str) -> String {
   text.replace(r"\", r"\\").replace("'", r"\'")
 }
+
+pub fn setup_builtin_modules(
+  js_runtime: &mut JsRuntime,
+) -> Result<(), AnyError> {
+  let module = "assert".to_string();
+  let script = format!(
+    r#"
+      const {} = globalThis.require('{}');
```

Comment:
require or globalThis.require throws `error: Uncaught ReferenceError: require is not defined`

Reply:
Apply suggestion from my previous comment to fix this problem